docs/core/compatibility/syslib0006.md

```diff
@@ -0,0 +1,36 @@
+---
+title: SYSLIB0006 warning
+description: Learn about the obsoletions that generate compile-time warning SYSLIB0006.
+ms.topic: reference
+ms.date: 10/20/2020
+---
+# SYSLIB0006: Thread.Abort is not supported
```

Comment:
Hmmm. I did a quick check in our conceptual docs and we have ~10 articles that mention `Thread.Abort`. Most seem to discourage its use. Should we link those articles back to this one?

Reply:
I logged an issue for this.